script/client/spring/application.properties

```diff
@@ -32,8 +32,8 @@ seata.client.undo.data-validation=true
 seata.client.undo.log-serialization=jackson
 seata.client.undo.log-table=undo_log
 seata.client.log.exceptionRate=100
-seata.service.vgroup-mapping=default
-seata.service.grouplist=127.0.0.1:8091
+seata.service.vgroup-mapping.my_test_tx_group=default
+seata.service.grouplist.default=127.0.0.1:8091
 seata.service.enable-degrade=false
 seata.service.disable-global-transaction=false
 seata.transport.shutdown.wait=3
```

Comment:
application.properties and application.yml underlineToCamel

Reply:
Not required.both camel and kebab style can be supported by springboot.